app/controllers/users_controller.rb

```diff
@@ -1,15 +1,19 @@
+# frozen_string_literal: true
+
 class UsersController < ApplicationController
   skip_before_action :authenticate_request
 
-  before_action only: :create, unless: -> { ActiveRecord::Type::Boolean.new.deserialize(ENV["SELF_REGISTRATION_AVAILABLE"]) } do
-    render json: { error: "Self registration is not available" }, status: :unprocessable_entity
+  before_action only: :create, unless: lambda {
```

Comment:
what's the reason to change `->` to `lambda` word?

Reply:
We can control this by the rule `Style/Lambda` cutomization:

```
Style/Lambda: Use the lambda method for multiline lambdas.
```
I have changed it to:
```
Style/Lambda:
  EnforcedStyle: literal
```
So, we will use `->` for any case from now.